### PR TITLE
Adds deploy docs

### PR DESCRIPTION
**Pull request by tallysmartins**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<img src="../web/public/images/logo.png" height="68" />
+<img src="https://github.com/elixir-bench/elixir-bench.github.io/blob/master/images/logo.png" height="68" />
```
Comment (PragTob) [inline README.md:1]: :+1:

File: README.md [hunk 2]
```diff
@@ -15,13 +15,34 @@ Benchmarks are running inside a docker container, so you need to have
 
 ## Deployment
 
-To build the release you can use `mix release`. The relese requires a `RUNNER_API_URL`, `RUNNER_API_KEY` and `RUNNER_API_USER`
-environment variables for communication with the API server.
+This project uses `distillery` for deployments. The relese requires a `RUNNER_API_URL`,
+`RUNNER_API_KEY` and `RUNNER_API_USER` environment variables for communication with
+the API server. Built releases are placed under `_build/prod/rel/runner`
```
Comment (PragTob) [inline README.md]: I think this just duplicates the information from the table below, which seems much better imo :) (aka I'd remove the mention of the envs here)

**Review by PragTob — COMMENTED** [review]
Small comment looks great otherwise

File: README.md (hunk 2 above)
Comment (tallysmartins) [reply README.md]: :100: removed!

Comment (PragTob) [reply README.md]: ? They're still up there? The place were you just removed them I actually liked :D

File: README.md [hunk 3]
```diff
@@ -32,10 +32,10 @@ The project requires the setup of the following environment variables:
 | `RUNNER_API_USER`|   username for authentication    |               test-runner              |
 | `RUNNER_API_KEY` | password key for authentication  |                  test                  |
 
-Start the application passing the required variables
+Set the variables and start the application with the command:
 
 ```bash
-RUNNER_API_URL=api.elixirbench.org/runner-api RUNNER_API_USER=test-runner RUNNER_API_KEY=test _build/prod/rel/runner/bin/runner foreground
```
Comment (PragTob) [inline README.md]: I actually liked having them here

File: README.md (hunk 2 above)
Comment (tallysmartins) [reply README.md]: oh sorry! bad reading :open_mouth:  I've just put it back! :+1:  :smile:

Comment (tallysmartins) [reply README.md]: removed now ;)

**Review by PragTob — APPROVED** [review]

**Pull request merged**